cegs_portal/search/views/v1/dna_features.py

```diff
@@ -215,6 +215,16 @@ def request_options(self, request):
     def get(self, request, options, data, chromo, start, end):
         features_paginator = Paginator(data, options["per_page"])
         feature_page = features_paginator.get_page(options["page"])
+        assembly_list = []
+        selected = False
+        all_assemblies = [GRCH38, GRCH37]
```

Comment:
since "all_assemblies" is now used in both views, it should be moved up to the constants section of the file in the form `ALL_ASSEMBLIES = [GRCH38, GRCH37]` and update both views to use the new constant.